api-client/package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "yarn run eslint",
     "lint:fix": "yarn run eslint --fix",
     "lint:ci": "yarn run eslint --quiet --format junit -o reports/lint/junit-lint.xml",
-    "ts:check": "yarn tsc --project tsconfig.json"
+    "ts:check": "tsc --build"
```

Comment:
Maybe it would still make sense to add the path to the tsconfig, as tsc --build by default [always selects the tsconfig in the current directory](https://www.typescriptlang.org/docs/handbook/project-references.html#tsc--b-commandline). So when running yarn workspace web ts:check I am not sure what tsconfig would be selected.